guides/user/canary/config/filter_templates/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: single
+title:  "Create and use filter templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+You can create sets of filters to apply against time series returned by your
+telemetry provider. These filters can also be parameterized, then hydrated by
+values provided in the [canary
+stage](/guides/user/canary/stage/#extended-params).
+
+## Things to keep in mind
+
+* Each filter template you create for a given canary configuration is available
+to each individual metric you add.
+
+* The query against your metrics is the metric type _plus_ selectors that refine
```

Comment:
Keep in mind that `selector` is a Stackdriver term.

Reply:
I forgot to mention resource type yesterday - the Stackdriver filter that Kayenta will compose is
```
metric.type="..." AND resource.type="..." AND {my-filter-template}
```

where `resource.type` is the resource type selected in the Canary Analysis stage config.

I don't know exactly what an example would look like for Prometheus but @duftler will know.

Reply:
I think was using it generically, but if Stackdriver uses it specifically and the others don't, yeah it'll be confusing. Is there a good generic term?

Reply:
I think it's a generic enough term, but will discuss with Dan on Monday (will also provide examples specific to the other service integrations).